cvat-ui/src/components/common/filters/job-filter-configuration-base.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Fields } from '@react-awesome-query-builder/antd';
+import asyncFetchUsers from 'components/resource-sorting-filtering/request-users';
+
+export const fields: Fields = {
```

Comment:
I am not sure we need a dedicated file

We may just use lodash.pick to get only specific filter properties from jobs page filter in quality control filter.